tweakable_encryptor_test.go

```diff
@@ -119,9 +119,20 @@ func (pes *testEncryptStream) encryptBlock(isFinal bool) error {
 	return nil
 }
 
-func (pes *testEncryptStream) init(version Version, sender BoxSecretKey, receivers []BoxPublicKey) error {
+func (pes *testEncryptStream) init(
+	version Version, sender BoxSecretKey, receivers []BoxPublicKey,
+	ephemeralKeyCreator EphemeralKeyCreator, rng encryptRNG) error {
+	if err := checkKnownVersion(version); err != nil {
```

Comment:
make this closer to encryptStream

Reply:
Unfortunate, I guess they diverged over time after this version got forked off?

Reply:
Yeah, I guess so. :(